app/tests/unit/validators/common.spec.js

```diff
@@ -0,0 +1,328 @@
+const crypto = require('crypto');
+const jestJoi = require('jest-joi');
+expect.extend(jestJoi.matchers);
+
+const { Permissions } = require('../../../src/components/constants');
+const { alphanumModel, truthyModel, uuidv4, uuidv4MultiModel, stringMultiModel, permCodeMultiModel } = require('../../../src/validators/common');
```

Comment:
With this many discrete attributes being imported, may be good to multi-line them for readability purposes (we're well past the 80 col character limit here)